build.sbt

```diff
@@ -7,29 +7,36 @@ description:= "handle outbound messages from salesforce to update zuora and iden
 
 version := "1.0"
 
-scalaVersion := "2.12.15"
+scalaVersion := "2.12.19"
 
 scalacOptions ++= Seq(
   "-deprecation",
   "-encoding", "UTF-8",
-  "-target:jvm-1.8",
+  "-release:8",
   "-Ywarn-dead-code"
 )
 
+enablePlugins(ScalaxbPlugin)
+
+Compile / scalaxb / scalaxbPackageName := "salesforce.soap"
+Compile / scalaxb / scalaxbGenerateDispatchClient := false // we don't need to use the 'dispatch' library
+
 val AwsVersion = "1.12.705"
 
 libraryDependencies ++= Seq(
   "com.amazonaws" % "aws-lambda-java-core" % "1.2.1",
   "com.amazonaws" % "aws-java-sdk-sqs" % AwsVersion,
   "com.amazonaws" % "aws-java-sdk-s3" % AwsVersion,
+  "org.scala-lang.modules" %% "scala-xml" % "2.2.0",
+  "org.scala-lang.modules" %% "scala-parser-combinators" % "2.4.0",
```

Comment:
`scala-xml` & `scala-parser-combinators` are required by the parsers generated by `scalaxb`.